mobile-ui/src/main/java/com/playone/mobile/ui/playone/PlayoneActivity.kt

```diff
@@ -25,15 +28,48 @@ class PlayoneActivity : BaseActivity() {
 
         viewModel = ViewModelProviders.of(this, viewModelFactory)
             .get(LoginViewModel::class.java)
+        viewModel.isProgressing.observe(this, Observer {
+            it?.takeIf { it }?.apply { showProgress() } ?: hideProgress()
```

Comment:
```kotlin
if (true == it) showProgress() else hideProgress()
```
It seems better! 
#### __OR__ using your `ifTrue{}` will be COOL when you finish `ifTrue()`.